### DISCUSSION TITLE
23.11.14 데일리 스크럼

### DISCUSSION THREAD

**Discussion by Jeonwoojae** [petit-a-petit/MIML#64] [Daily Scrum]
.

**Comment by Jeonwoojae** [comment]
- 오늘은 어떤 일을 했나요?
  - 노래 저장 기능 구현 방법 결정
- 내일은 어떤 일을 하나요?
  - 노래 저장 기능 구현

**Comment by minju412** [comment]
- 오늘은 어떤 일을 했나요?
    - `SimpleAuthFilter` 반영
    - 이메일, 패스워드로 스포티파이 로그인 시 에러 발생..
- 내일은 어떤 일을 하나요?
    - 에러 해결
